cloudinit/cmd/devel/logs.py

```diff
@@ -22,11 +24,46 @@
 CLOUDINIT_RUN_DIR = "/run/cloud-init"
 
 
+class ApportFile(NamedTuple):
+    path: str
+    label: str
+
+
+INSTALLER_APPORT_SENSITIVE_FILES = [
+    ApportFile(
+        "/var/log/installer/autoinstall-user-data", "AutoInstallUserData"
+    ),
+    ApportFile("/autoinstall.yaml", "AutoInstallYAML"),
+    ApportFile("/etc/cloud/cloud.cfg.d/99-installer.cfg", "InstallerCloudCfg"),
+]
+
+INSTALLER_APPORT_FILES = [
+    ApportFile("/var/log/installer/ubuntu_desktop_installer.log", "UdiLog"),
+    ApportFile(
+        "/var/log/installer/subiquity-server-debug.log", "SubiquityLog"
+    ),
+    ApportFile(
+        "/var/log/installer/subiquity-client-debug.log", "SubiquityClientLog"
+    ),
+    ApportFile("/var/log/installer/curtin-install.log", "CurtinLog"),
+    ApportFile(
+        "/var/log/installer/subiquity-curtin-install.cfg", "CurtinConfig"
```

Comment:
With the version of Subiquity released for kinetic, this file was split into different files:

`/var/log/installer/curtin-install/subiquity-initial.conf`
`/var/log/installer/curtin-install/subiquity-partitioning.conf`
`/var/log/installer/curtin-install/subiquity-extract.conf`
`/var/log/installer/curtin-install/subiquity-curthooks.conf`

Reply:
If you would like to keep compatibility with older versions, it seems that the right name for this file was `subiquity-curtin-install.conf` instead of `.cfg` :)